exercises/triangle/README.md

```diff
@@ -4,9 +4,7 @@ Determine if a triangle is equilateral, isosceles, or scalene.
 
 An _equilateral_ triangle has all three sides the same length.
 
-An _isosceles_ triangle has at least two sides the same length. (It is sometimes
-specified as having exactly two sides the same length, but for the purposes of
-this exercise we'll say at least two.)
+An _isosceles_ triangle has at least two sides the same length.
```

Comment:
Why does this need to be updated?

Reply:
(In addition, note that any desired deviations need to be handled at the level of the exercise README template file to survive `configlet` README regeneration.)